model/src/main/kotlin/MergedAnalyzerResult.kt

```diff
@@ -20,6 +20,8 @@
 package com.here.ort.model
 
 import com.fasterxml.jackson.annotation.JsonProperty
+import com.here.ort.utils.normalizePath
```

Comment:
Nit: Blank line before this one (despite both starting with "com").

Reply:
Done.